src/windows/process.rs

```diff
@@ -131,18 +136,22 @@ unsafe fn get_process_user_id(
 
     let mut size = 0;
 
-    if GetTokenInformation(*token, TokenUser, null_mut(), 0, &mut size) == 0 {
-        let err = GetLastError();
-        if err != ERROR_INSUFFICIENT_BUFFER {
+    let ret = GetTokenInformation(*token, TokenUser, None, 0, &mut size);
+    if let Err(err) = ret {
+        if err.code() != ERROR_INSUFFICIENT_BUFFER.to_hresult() {
```

Comment:
Since you don't use `ret`, what do you think about:

```suggestion
    if let Err(err) = GetTokenInformation(*token, TokenUser, None, 0, &mut size);
        if err.code() != ERROR_INSUFFICIENT_BUFFER.to_hresult() {
```

?